microsoftgraph/microsoft-graph-explorer-v4
# Feature: show delta link

## Overview
Adds a banner to display the delta link.

Reorganises the code so that the code is clean and without repetition.

Closes #913 

### Demo

With nextLink
![image](https://user-images.githubusercontent.com/58787602/120663564-cc7dd500-c492-11eb-97dc-5b4fc73009bb.png)

After following the next link, you get the delta link

![image](https://user-images.githubusercontent.com/58787602/120663647-e0c1d200-c492-11eb-9312-cf464a7dabcf.png)


### Notes

Optional. Ancillary topics, caveats, alternative strategies that didn't work out, anything else.

## Testing Instructions

* Run this query 
  [https://graph.microsoft.com/v1.0/me/calendarView/delta?startdatetime=2016-12-01T00:00:00Z&enddatetime=2016-12-30T00:00:00Z](https://graph.microsoft.com/v1.0/me/calendarView/delta?startdatetime=2016-12-01T00:00:00Z&enddatetime=2016-12-30T00:00:00Z)
* Notice the banner with the link
* Follow the clickable link 
* Notice the delta link banner with the link
